Add dmca.json and check if post contains a banned author or id

jm90m opened the pull request:
Fixes #1662 .

Changes:
- Check post to see if it contains a banned author or post id

bonustrack commented on src/common/constants/dmca.json:
```diff
@@ -0,0 +1,4 @@
+{
+  "authors": [],
+  "post_ids": [13359110, 29061561]
```
it would be easier to have the posts with `author/permlink`, is authors id based too?

jm90m replied:
no it isnt, but yeah i can change it to author/permlink

Sekhmet replied:
`authors` are usernames.

jm90m commented:
changed post_ids to post_urls, and changed check to check for the url that you mentioned.
```
....  
const postURL = `${post.author}/${post.permlink}`;

return _.includes(bannedAuthors, post.author) || _.includes(bannedPostURLs, postURL);
```

@bonustrack

bonustrack commented on src/common/constants/dmca.json:
```diff
@@ -0,0 +1,4 @@
+{
+  "authors": [],
+  "post_urls": ["cracked-games/ltxziiz9", "bleh773/2ryjej-on-off"]
```
Can you name it `posts` and put them on multiple line?

jm90m replied:
yup just did
